query_optimizer/PhysicalGenerator.cpp

```diff
@@ -108,6 +109,7 @@ P::PhysicalPtr PhysicalGenerator::generateInitialPlan(
 P::PhysicalPtr PhysicalGenerator::optimizePlan() {
   std::vector<std::unique_ptr<Rule<P::Physical>>> rules;
   rules.emplace_back(new PruneColumns());
+  rules.emplace_back(new PushDownLowCostDisjunctivePredicate());
```

Comment:
The three rules blocks from `PushDownLowCostDisjunctivePredicate` to `ReorderColumns` can be arranged in any order.

`PushDownLowCostDisjunctivePredicate` will not generate unnecessary `Selection`s. And currently in the physical generate there's no rule that fuses `Selection`s.

I add a comment to indicate that currently new rules should better be added before `AttachLIPFilters` because rules after that needs extra handling of `LIPFilterConfiguration` for transformed nodes.